ui/src/services/groups/groups.service.ts

```diff
@@ -134,6 +134,12 @@ export class GroupsService extends BaseService {
         return this.httpGet<ArtifactMetaData>(endpoint);
     }
 
+    public getLatestArtifact(groupId: string|null, artifactId: string): Promise<any> {
+        groupId = this.normalizeGroupId(groupId);
+        let endpoint: string = this.endpoint("/v2/groups/:groupId/artifacts/:artifactId", { groupId, artifactId });
+        return this.httpGet<any>(endpoint)
+    }
```

Comment:
This could be implemented as:

```
    public getLatestArtifact(groupId: string|null, artifactId: string): Promise<any> {
        return this.getArtifactContent(groupId, artifactId, "latest");
    }
```